INSTALL.rst

```diff
@@ -75,12 +98,17 @@ Add demo records
    (inspirehep)$ inspirehep migrator populate -f inspirehep/demosite/data/demo-records.xml.gz
 
 
+.. note::
+
+    Remember to keep `honcho` running in a separate window.
+
+
 Adding a single record with CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: console
 
-   (inspirehep)$ demouuid = $(dojson do -l marcxml -i inspirehep/demosite/data/sample.xml hep | inspirehep records create)
+   (inspirehep)$ demouuid = $(dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep | inspirehep records create)
```

Comment:
Does this index the record? we can also run the migrator command pointing to a marcxml with one record.